virttest/utils_test/libvirt.py

```diff
@@ -1499,6 +1499,31 @@ def check_state():
         return utils_misc.wait_for(check_state, timeout)
 
 
+def check_status_output(status, output='',
+                        expected_fails=[],
+                        skip_if=[],
+                        any_error=False,
+                        expected_match=[]):
+    """
+    Proxy function to call check_result for commands run in vm session.
+    :param status: Exit status (used as CmdResult.exit_status)
```

Comment:
Please add one blank line between description and params. ^^

Reply:
Sorry, done :)